test/e2e/page-objects/pages/dialog/terms-of-use-update-modal.ts

```diff
@@ -44,7 +44,10 @@ class TermsOfUseUpdateModal {
       5000,
     );
     await this.driver.clickElement(this.termsOfUseCheckbox);
-    await this.driver.clickElementAndWaitToDisappear(this.acceptButton);
+    await this.driver.clickElementAndWaitToDisappear(
+      this.acceptButton,
+      5000,
+    );
```

Comment:
while fixing the main issue, I saw another point of flakiness, where sometimes this element has not yet disappeared (as the default is 2 seconds) so I'm increasing a bit the buffer here, to prevent failing in this step


![Screenshot from 2025-06-11 16-00-48](https://github.com/user-attachments/assets/15e6eccb-3ec7-4ecf-80e9-45767536bf5d)

![test-failure-screenshot-2](https://github.com/user-attachments/assets/ffcffd62-fd65-4a72-bc74-97c42d246125)